_config.yml

```diff
@@ -18,8 +18,8 @@ description:
   en: Working in the open! Our work in progress, list of reference material and presentations.
   fr: Travailler ouvertement! Nos travaux en cours, liste de documents de références et présentations.
 sectionsList:
-  en: ["About us", "Work In Progress", "IT Picture", "Trends", "Under Review", "Archives"]
-  fr: ["À propos de nous", "Travaux en cours", "Image de la TI", "Tendances", "En révision", "Archives"]
+  en: ["About us","IT Picture", "Trends", "Under Review", "Work In Progress", "Archives"]
```

Comment:
What is Trends meant to be?

Reply:
A section to list trends such as OneGC, OneTeamGov, etc. Nothing tagged under that section so not visible at the moment